docs/source/reference/config.rst

```diff
@@ -40,6 +40,24 @@ Available fields and semantics:
     - gcp
     - kubernetes
 
+  docker:
+    # Additional Docker run options (optional).
+    #
+    # When image_id: docker:<docker_image> is used in a task YAML, additional
+    # run options for starting the Docker container can be specified here.
+    # The default run options are:
+    #   --net=host
+    #   --cap-add=SYS_ADMIN
+    #   --device=/dev/fuse
+    #   --security-opt=apparmor:unconfined
+    #
+    # This field can be useful for mounting volumes and other advanced Docker
+    # configurations. The following is an example option for allowing running
+    # Docker inside Docker.
+    #   sky launch --cloud aws --image-id docker:continuumio/miniconda3 "apt update; apt install -y docker.io; docker run hello-world"
+    run_options:
+      - -v /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Should we do a multi-arg example here? E.g., it's not clear if multiple args should be specified as list or a long str
```suggestion
    # This field can be useful for mounting volumes and other advanced Docker
    # configurations. The following is an example option for allowing running
    # Docker inside Docker and increase the size of /dev/shm:
    #   sky launch --cloud aws --image-id docker:continuumio/miniconda3 "apt update; apt install -y docker.io; docker run hello-world"
    run_options:
      - -v /var/run/docker.sock:/var/run/docker.sock
      - --shm-size=2g
```

Reply:
This is a great point! Made the instruction a bit more concrete. : )